### ISSUE TITLE
Error in Running the Code

### ISSUE THREAD

**Issue by zakiazaman34** [zhiruiluo/abode_net#1]
Hi,
While running the code, it raises the following error:
 **_/abode_net-main/src/models/base_module.py", line 19, in <module>
    from transformers import (ModuleNotFoundError: No module named 'transformers'_**

**Comment by zhiruiluo** [comment]
Hi @zakiazaman34 thanks for pointing this out. Yeah, the dependency of transformers should be put into the ```requirement.txt```.
```python
from transformers import (
    get_cosine_schedule_with_warmup,
    get_cosine_with_hard_restarts_schedule_with_warmup,
)
```
The transformer simply provides scheduler for the base trainer.
You could simply run```pip install transformers``` for a quick fix.

If you would like to contribute a PR, please do so.

**Comment by zhiruiluo** [comment]
Close this by the fix #2.

**Comment by zakiazaman34** [comment]
Thank you!!
